misc/config_tools/data/tgl-vecow-spc-7100-Corei7/shared.xml

```diff
@@ -0,0 +1,213 @@
+      <RDT>
+        <RDT_ENABLED>n</RDT_ENABLED>
+        <CDP_ENABLED>n</CDP_ENABLED>
+        <VCAT_ENABLED>n</VCAT_ENABLED>
+        <CLOS_MASK>0xfffff</CLOS_MASK>
+        <CLOS_MASK>0xfffff</CLOS_MASK>
+        <CLOS_MASK>0xfffff</CLOS_MASK>
+        <CLOS_MASK>0xfffff</CLOS_MASK>
+        <CLOS_MASK>0xfffff</CLOS_MASK>
+        <CLOS_MASK>0xfffff</CLOS_MASK>
+        <CLOS_MASK>0xfffff</CLOS_MASK>
+        <CLOS_MASK>0xfffff</CLOS_MASK>
+      </RDT>
+      <NVMX_ENABLED>n</NVMX_ENABLED>
+      <HYPERV_ENABLED>y</HYPERV_ENABLED>
+      <IOMMU_ENFORCE_SNP>n</IOMMU_ENFORCE_SNP>
+      <ACPI_PARSE_ENABLED>y</ACPI_PARSE_ENABLED>
+      <L1D_VMENTRY_ENABLED>n</L1D_VMENTRY_ENABLED>
+      <MCE_ON_PSC_DISABLED>n</MCE_ON_PSC_DISABLED>
+      <IVSHMEM>
+        <IVSHMEM_ENABLED>n</IVSHMEM_ENABLED>
+        <IVSHMEM_REGION/>
+      </IVSHMEM>
+    </FEATURES>
+    <MEMORY>
+      <STACK_SIZE>0x2000</STACK_SIZE>
+      <PLATFORM_RAM_SIZE>0x400000000</PLATFORM_RAM_SIZE>
+    </MEMORY>
+    <CAPACITIES>
+      <MAX_VM_NUM>16</MAX_VM_NUM>
+      <IOMMU_BUS_NUM>0x100</IOMMU_BUS_NUM>
+      <MAX_IOAPIC_NUM>1</MAX_IOAPIC_NUM>
+      <MAX_PCI_DEV_NUM>96</MAX_PCI_DEV_NUM>
+      <MAX_IOAPIC_LINES>120</MAX_IOAPIC_LINES>
+      <MAX_PT_IRQ_ENTRIES>256</MAX_PT_IRQ_ENTRIES>
+      <MAX_MSIX_TABLE_NUM/>
+      <MAX_EMULATED_MMIO>16</MAX_EMULATED_MMIO>
+    </CAPACITIES>
+    <MISC_CFG>
+      <GPU_SBDF>0x00000010</GPU_SBDF>
+    </MISC_CFG>
+  </hv>
+  <vm id="0">
+    <vm_type>SERVICE_VM</vm_type>
+    <name>ACRN_Service_VM</name>
+    <guest_flags>
+      <guest_flag>0</guest_flag>
+    </guest_flags>
+    <clos>
+      <vcpu_clos>0</vcpu_clos>
+    </clos>
+    <memory>
+      <start_hpa>0</start_hpa>
+      <size>0x20000000</size>
+    </memory>
+    <os_config>
+      <name>ACRN Service OS</name>
+      <kern_type>KERNEL_BZIMAGE</kern_type>
+      <kern_mod>Linux_bzImage</kern_mod>
+      <ramdisk_mod/>
+      <bootargs>SERVICE_VM_OS_BOOTARGS</bootargs>
+    </os_config>
+    <legacy_vuart id="0">
+      <type>VUART_LEGACY_PIO</type>
+      <base>SERVICE_VM_COM1_BASE</base>
+      <irq>SERVICE_VM_COM1_IRQ</irq>
+    </legacy_vuart>
+    <legacy_vuart id="1">
+      <type>VUART_LEGACY_PIO</type>
+      <base>SERVICE_VM_COM2_BASE</base>
+      <irq>SERVICE_VM_COM2_IRQ</irq>
+      <target_vm_id>1</target_vm_id>
+      <target_uart_id>1</target_uart_id>
+    </legacy_vuart>
+    <legacy_vuart id="2">
+      <type>VUART_LEGACY_PIO</type>
+      <base>SERVICE_VM_COM3_BASE</base>
+      <irq>SERVICE_VM_COM3_IRQ</irq>
+      <target_vm_id>2</target_vm_id>
+      <target_uart_id>2</target_uart_id>
+    </legacy_vuart>
+    <console_vuart id="0">
+      <base>INVALID_PCI_BASE</base>
+    </console_vuart>
+    <communication_vuart id="1">
+      <base>INVALID_PCI_BASE</base>
+      <target_vm_id>1</target_vm_id>
+      <target_uart_id>1</target_uart_id>
+    </communication_vuart>
+    <pci_devs>
+      <pci_dev/>
+    </pci_devs>
+      <rootfs>/dev/sda3</rootfs>
```

Comment:
The whole board_private section needs to be deleted, from <board_private> to </board_private>

Reply:
fix